packages/edit-site/src/components/resizable-frame/index.js

```diff
@@ -171,7 +171,12 @@ function ResizableFrame( {
 		event.preventDefault();
 
 		const step = 20 * ( event.shiftKey ? 5 : 1 );
-		const delta = step * ( event.key === 'ArrowLeft' ? 1 : -1 );
+		const delta =
+			step *
+			( ( ! isRTL() && event.key === 'ArrowLeft' ) ||
+			( isRTL() && event.key === 'ArrowRight' )
+				? 1
+				: -1 );
```

Comment:
I’d be tempted to use multiplication and flip the delta again just in hopes it reads better but it’s fine as is.
```suggestion
			( event.key === 'ArrowLeft' ? 1 : -1 ) *
			( isRTL() ? -1 : 1 );
```